crates/cargo-test-support/src/registry.rs

```diff
@@ -558,6 +558,7 @@ pub struct Dependency {
     package: Option<String>,
     optional: bool,
     default_features: bool,
+    public: bool,
```

Comment:
Feel free to break this out into its own PR for us to merge immiedately

Reply:
Ok